services/core-web/src/components/Forms/AddTailingsForm.js

```diff
@@ -1,21 +1,38 @@
 import React from "react";
 import PropTypes from "prop-types";
+import { compose } from "redux";
+import { connect } from "react-redux";
 import { Field, reduxForm } from "redux-form";
 import { Form } from "@ant-design/compatible";
 import "@ant-design/compatible/assets/index.css";
 import { Button, Col, Row, Popconfirm } from "antd";
-import { required } from "@common/utils/Validate";
+import { required, maxLength, number, lat, lon } from "@common/utils/Validate";
+import {
+  getConsequenceClassificationStatusCodeDropdownOptions,
+  getTSFOperatingStatusCodeDropdownOptions,
+} from "@common/selectors/staticContentSelectors";
 import { resetForm } from "@common/utils/helpers";
 import RenderField from "@/components/common/RenderField";
+import PartySelectField from "@/components/common/PartySelectField";
+import RenderSelect from "@/components/common/RenderSelect";
 import * as FORM from "@/constants/forms";
+import CustomPropTypes from "@/customPropTypes";
 
 const propTypes = {
   handleSubmit: PropTypes.func.isRequired,
   closeModal: PropTypes.func.isRequired,
   title: PropTypes.string.isRequired,
   submitting: PropTypes.bool.isRequired,
+  consequenceClassificationStatusCodeOptions: PropTypes.arrayOf(CustomPropTypes.dropdownListItem)
+    .isRequired,
+  TSFOperatingStatusCodeOptions: PropTypes.arrayOf(CustomPropTypes.dropdownListItem).isRequired,
 };
 
+const boolData = [
+  { value: "false", label: "No" },
+  { value: "true", label: "Yes" },
+];
```

Comment:
Ant design select components do not allow `false` as a value, had to cast to a string in order to use the select with booleans.